src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java

```diff
@@ -440,16 +441,18 @@ public void visit(Column tableColumn) {
         buffer.append(OPTIONAL_WHITE_SPACE);
         final Table table = tableColumn.getTable();
         String tableName = "";
-        if (table != null && table.getFullyQualifiedName() != null) {
+        if (table != null && !table.getFullyQualifiedName().isEmpty()) {
             tableName += table.getFullyQualifiedName();
             if (table.getAlias() != null) {
-                tableName += table.getAlias().getName();
+                tableName += table.getAlias().toString();
             }
-        }
-
-        if (!tableName.isEmpty()) {
-            buffer.append(RegExGenerator.useSpellingMistake(this.tableNameSpellingMistake, tableName));
-            buffer.append('.');
+            tableName =
+                    "(?:"
+                    + RegExGenerator.useSpellingMistake(this.columnNameSpellingMistake, tableName)
```

Comment:
Arent there now to 2 settings mixed up? Column and TableNameSpelling?

Reply:
oh my fault